doc/assets/expr.dot

```diff
@@ -0,0 +1,5 @@
+graph {
+    ID_plus -- 3 [ label = "sub[0]" ];
+    ID_plus -- 5 [ label = "sub[1]" ];
+    ID_plus -- ID_integer [ label = "named_sub[ID_type]" ];
+}
```

Comment:
It seems elsewhere dot graphs are just included inline, so maybe this can be done here as well? Having a directory for a five-line file seems like overkill to me?

Reply:
The directory is already there and holds similar sorts of graph. Adding this inline would degrade the readability of the code I think.

Reply:
Ok, sorry, I wasn't aware. Yet that might be enough of a reason to get others to do the same? A lot more graphviz diagrams have been introduced in recent PRs, with yours being the only one to place the graph in this folder...